accounts.go

```diff
@@ -342,3 +347,66 @@ func applyUpdateMask(mask *field_mask.FieldMask, msg protoreflect.ProtoMessage,
 	fmutils.Filter(msg, allowedFields)
 	return nil
 }
+
+func (srv *accountsAPI) AuthenticateGoogle(ctx context.Context, in *accountsv1.AuthenticateGoogleRequest) (*accountsv1.AuthenticateGoogleResponse, error) {
+
+	code := in.Code
```

Comment:
You're using only once the in.Code, I think you don't need a variable to describe what it is.

Reply:
Yes, I remove this variable